easybuild/tools/filetools.py

```diff
@@ -718,9 +718,10 @@ def apply_patch(patch_file, dest, fn=None, copy=False, level=None):
         _log.debug("Using specified patch level %d for patch %s" % (level, patch_file))
 
     patch_cmd = "patch -b -p%s -i %s" % (level, apatch)
-    result = run.run_cmd(patch_cmd, simple=True, path=adest)
+    result = run.run_cmd(patch_cmd, simple=True, path=adest, log_ok=False)
+
     if not result:
-        raise EasyBuildError("Patching with patch %s failed", patch_file)
+        raise EasyBuildError("Couldn't apply patch")
```

Comment:
keep mentioning name of patch file, and maybe also grab the output from the `patch` command and include it in the error message (you'll need to use `simple=False` in `run_cmd` and do `out, ec = run_cmd(...)`